www/js/app.js

```diff
@@ -153,11 +153,37 @@ function resizeIFrame () {
         }, 100);
     }
 }
+
+/**
+ * Resize the IFrame height, so that it fills the whole available height in the window
+ */
+function resizeLibrary () {
```

Comment:
Hmm, this function seems to be a direct copy of `resizeIframe()`, with some modifications. This is not an efficient way to repurpose that code. You should instead call the exisitng `resizeIframe()` funciton and if you need to run specific code for the library within it, you should pass a string like `library` (or better, the id of the iframe to resize) to it, so you can detect we're resizing the library. That should be pretty easy to do.